components/dropzone/dropzone.jsx

```diff
@@ -1,73 +1,93 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { classnames } from '../../helpers/component';
+
 import './dropzone.sass';
 
 const CLASSNAMES = {
-    default: 'dropBorder-default',
-    error: 'dropBorder-error',
-    success: 'dropBorder-success'
+  default: 'dropBorder-default',
```

Comment:
Hello @Yiin , thank you for the feedback. I switched the indent to use 4 spaces instead of 2 and I set the default value directly in the destructuring assignment.

I had to set the default for the function differently since I was getting an error in the editor.